parm/atm/obs/config/lgetkf_amsua_n19.yaml

```diff
@@ -6,11 +6,11 @@ obs space:
   obsdatain:
     engine:
       type: H5File
-      obsfile: !ENV ./obs/${OPREFIX}amsua_n19.${PDY}${cyc}.nc4
+      obsfile: ./obs/$(OPREFIX)amsua_n19.{{ current_cycle | to_YMDH }}.nc4
```

Comment:
why not $(DATA) here? I know it was not like that before but should it be?

Reply:
I can't find the issue or email but I believe inclusion of `$(DATA)` in `obsfile` occasionally caused fv3jedi executables to fail because the number of characters in the `$(DATA)/.....` string exceeded the maximum length allowed by the code.   Has this limitation been removed?

Reply:
hmm, ok that is possible. I knew it did that for backgrounds with an FMS IO limitation, but didn't know about it for IODA.

Reply:
I'll resolve for now, let's remember when we discuss naming conventions for input files to include relative/full paths to the obs in there.